pytorch_to_returnn/torch/nn/modules/operator.py

```diff
@@ -61,13 +61,16 @@ def create_returnn_layer_dict(self, low, high, size, dtype=None) -> Dict[str, An
     naming = Naming.get_instance()
     call = naming.module_call_stack[-1]
     assert call.module.module is self
-    # add dependency to get complete in feed_dict in Module.make_output_tensor_from_returnn
-    call.inputs_tensor_deps.append(naming.inputs[0])
     source = set()
     for sz in size:
       if isinstance(sz, Tensor):
-        if naming.tensors[sz].is_size_value.originating_tensor is not None:
+        assert naming.tensors[sz].is_size_value is not None
```

Comment:
I think the code becomes easier if you first assign `naming.tensors[sz]` to another var, like:
```
tensor_entry = naming.tensors[sz]
```